packages/search-ui-app-search-connector/src/__tests__/AppSearchAPIConnector.test.js

```diff
@@ -345,17 +345,32 @@ describe("AppSearchAPIConnector", () => {
         current: 2,
         searchTerm: "searchTerm"
       };
-      const beforeSearchCall = (queryOptions, next) =>
-        next({
-          ...queryOptions,
+
+      const queryConfig = {
+        sortDirection: "desc",
+        sortField: "name",
+        resultsPerPage: 5
+      };
+
+      const beforeSearchCall = (options, next) => {
+        // Remove sort_direction and sort_field
+        // eslint-disable-next-line no-unused-vars
+        const { sort, ...rest } = options;
```

Comment:
just curious: would a dev be able to `delete options.sort` instead? or would they not want to do that since it's mutating the original `options` obj?

Reply:
@constancecchen I *think* they could do that without any adverse side effects.